sponsorblock.lua

```diff
@@ -383,17 +398,23 @@ function file_loaded()
     last_skip = {uuid = "", dir = nil}
     chapter_cache = {}
     local video_path = mp.get_property("path")
+    mp.msg.debug("Path: " .. video_path)
+    local video_referer = string.match(mp.get_property("http-header-fields"), "Referer:([^,]+)")
```

Comment:
Add `or ""`, currently it crashes on the next line when referer isn't present.

Reply:
Still crashes, use `local video_referer = string.match(mp.get_property("http-header-fields", ""), "Referer:([^,]+)") or ""`

Reply:
Sorry about that, I have now set up my environment such that I can also test the non-SVP case - it shouldn't crash anymore.